GNUmakefile

```
@@ -204,7 +204,11 @@ check: ## Lint the source code
 
 	@echo "==> Check proto files are in-sync..."
 	@$(MAKE) proto
-	@if (git status | grep -q .pb.go); then echo the following proto files are out of sync; git status |grep .pb.go; exit 1; fi
+	@if (git status -s | grep -q .pb.go); then echo the following proto files are out of sync; git status -s | grep .pb.go; exit 1; fi
+
+	@echo "==> Check format of jobspecs and HCL files..."
+	@$(MAKE) hclfmt
+	@if (git status -s | grep -q -e '\.hcl$$' -e '\.nomad$$'); then echo the following HCL files are out of sync; git status -s | grep -e '\.hcl$$' -e '\.nomad$$'; exit 1; fi
```

I think we only need a single `$` here.

```suggestion
	@if (git status -s | grep -q -e '\.hcl$' -e '\.nomad$'); then echo the following HCL files are out of sync; git status -s | grep -e '\.hcl$' -e '\.nomad$'; exit 1; fi
```

---

I thought so at first but when I tested it, I realized `Make` wants you to escape the `$`, so it was being ignored without that and the results were wrong.

---

:til: Thanks for your testing :).  Ship it!